### DISCUSSION TITLE
8月8日付款，收不到验证码，到现在依然无法使用，请管理员尽快解决一下

### DISCUSSION THREAD

**Discussion by juzhier** [getlantern/lantern-forum-cn#873] [Q&A]
8月8日买的专业版，支付账号邮箱leecx23184260@gmail.com ，收不到验证码，一直没法使用，前几天联系给了我一个验证码106329，但是输入无效，现在应该怎么办，如果有新的验证码不要发我gmail的邮箱，请直接留言给我。
付费凭证如下：
![微信图片_20240813083611](https://github.com/user-attachments/assets/f1b241c2-48c0-471c-aa7a-14dc8d0c304a)
![微信图片_20240918205654](https://github.com/user-attachments/assets/9e2460b2-7a03-4bad-82e3-03ed014e9c35)

**Comment by MaxMKMax** [comment]
@juzhier 好的. 已经帮您提交问题上去了. 请您放心, 会尽快给您解决问题!

**Comment by greySkiesMarsh** [comment]
Hello @juzhier 

I've cleared out all linking data from your account. Please try to log in again to see if that solved the error.
